uberfire-services/uberfire-services-api/src/main/java/org/guvnor/common/services/backend/metadata/MetadataServerSideService.java

```diff
@@ -28,4 +28,8 @@
     Metadata getMetadata(final Path resource);
 
     List<String> getTags(final Path resource);
+
+    Path saveMetadata(final Path resource,
+                      final Metadata metadata,
+                      final String comment);
```

Comment:
Please add a javadoc,  I think for newcomers can be unclear if this method save really just metadata or also `path`.